### ISSUE TITLE
Unable to download certain tracks - Territory Restricted (But it's not really restricted)

### ISSUE THREAD

**Issue by zhordon** [unspok3n/beatportdl#1]
Just stumbled across this project and find it quite interesting and useful.

I've stumbled across a track/release that appears as Territory Restricted, but when logged into Beatport website and using the account on actual DJ hardware the track downloads/streams just fine.

Release: https://www.beatport.com/release/pegasus-extended-mix/4561083
Track: https://www.beatport.com/track/pegasus/19007718

Website:
![image](https://github.com/user-attachments/assets/36651353-3cab-49c5-8b39-39405018d1d3)

Denon Prime 4+:

![image](https://github.com/user-attachments/assets/e765604f-a7fd-415a-bfd3-9e6861485423)
![image](https://github.com/user-attachments/assets/ba6761c8-e4d4-4a5c-b2ac-1eb302d3639f)
![image](https://github.com/user-attachments/assets/9b11bb57-dc5b-490e-a387-eedc96a16cf8)

While using beatportdl:
```
Enter track or release link: https://www.beatport.com/track/pegasus/19007718
fetch track: request failed with status code: 403 - Territory Restricted.

Press enter to exit
```

Is there a potential fix for this?

Thanks again for this project.

**Comment by unspok3n** [comment]
Hi! Glad you found this project useful!

In the photos you show https://www.beatport.com/track/pegasus/19007718 on the website and downloading https://www.beatport.com/track/pegasus-how-we-know/19007876 on the DJ controller. That's 2 different tracks.

There's indeed something wrong with https://www.beatport.com/track/pegasus/19007718 cause its available through the webapp but territory restricted when using public API, but its up to beatport to fix that.

**Comment by zhordon** [comment]
You are absolutely correct, I just took another look at the links and the hardware, and it was the wrong track on the hardware.

I created a test playlist with the track in question and it fails to show up on the DJ hardware. When refreshing the webpage it shows as a full track to stream, but when you play it cuts down to a limited section. 

So this appears to be normal behaviour. I verified with a couple of other tracks with the preview bar and same result. The messaging should be clearer from the API in that it's not streamable rather then Territory Restricted.
